### PR TITLE
First task

### PR DESCRIPTION
**Pull request by Shake-Ghairbyan**
_Added ReadMe, .gitignore fille, Main class._

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
```
Comment (chandranik90) [inline .gitignore]: out folder is generated by Idea after running. It's also necessary to ignore

**Review by chandranik90 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Shake-Ghairbyan) [reply .gitignore]: Didn't notice it this time. Will do.

> out folder is generated by Idea after running. It's also necessary to ignore

Comment (Shake-Ghairbyan) [reply .gitignore]: But "out" package is not visible in my directory.

> out folder is generated by Idea after running. It's also necessary to ignore

Comment (chandranik90) [reply .gitignore]: Just run your main method, and it will be generated

Comment (Shake-Ghairbyan) [reply .gitignore]: Actually, I have run it, please see the output below.
```
/Library/Java/JavaVirtualMachines/jdk-15.0.1.jdk/Contents/Home/bin/java -javaagent:/Applications/IntelliJ IDEA CE.app/Contents/lib/idea_rt.jar=59762:/Applications/IntelliJ IDEA CE.app/Contents/bin -Dfile.encoding=UTF-8 -classpath /Users/shake/Library/Caches/JetBrains/IdeaIC2020.3/compile-server/epam_java_automation_24211984/scratches/out Main
Hello world!

Process finished with exit code 0
```
But the `out/` folder was not generated.

Nevertheless, I have added it to the `.gitignore` file in 75bf09a

Comment (chandranik90) [reply .gitignore]: ok

**Review by chandranik90 — APPROVED** [review]

**Pull request merged**